### PR TITLE
Add missing fields

### PR DESCRIPTION
**Pull request by rqphqel**
# Description

#### Dependencies

- Bump [jsoniter](https://github.com/json-iterator/go) to 1.1.12
- Bump [testify](https://github.com/stretchr/testify) to 1.7.0

#### Temporary fixes

- Remove `GenreIDs`, `ID` and `VoteCount` from `CreditsDetails` and `FindByID`

These fields should be converted to `int64`.\
Non-breaking because `GetCreditDetails` and `GetFindByID` (person) never worked to begin with.

#### CertificationMovie
- Add `CA_QC`, `DK`, `HU`, `IT`, `LT`, `MY`, `NO`, `RU` and `SE` (not documented)

#### CertificationTV
- Add `CA_QC`, `ES`, `HU`, `LT`, `NL`, `PH`, `PT` and `SK` (not documented)

#### ConfigurationCountries

- Add `NativeName` (not documented)

#### CreditsDetails

- Add `Adult`, `OriginalTitle`, `ReleaseDate`, `Title` and `Video` (movie only, documented [here](https://developers.themoviedb.org/3/movies/get-movie-details))
- Add _omitempty_ option to `FirstAirDate`, `Name`, `OriginalName`, `OriginCountry`, `Episodes` and `Seasons` (TV only)

#### FindByID / ListDetails

- Add `FirstAirDate`, `Name`, `OriginalName` and `OriginCountry` (TV only, documented [here](https://developers.themoviedb.org/3/find/find-by-id) / not documented)
- Add _omitempty_ option to `Adult`, `OriginalTitle`, `ReleaseDate`, `Title` and `Video` (movie only)

# Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Tested locally.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: credits.go [hunk 1]
```diff
@@ -51,25 +56,25 @@ type CreditsDetails struct {
...
-			VoteCount        int64    `json:"vote_count"`
-			Popularity       float32  `json:"popularity"`
-			MediaType        string   `json:"media_type"`
-			OriginalName     string   `json:"original_name,omitempty"`
-			Name             string   `json:"name,omitempty"`
-			FirstAirDate     string   `json:"first_air_date,omitempty"`
-			OriginCountry    []string `json:"origin_country,omitempty"`
+			Adult        bool   `json:"adult,omitempty"`
+			BackdropPath string `json:"backdrop_path"`
+			// GenreIDs         []int64  `json:"genre_ids"` // FIXME: -> []float32
```
Comment (cyruzin) [inline credits.go:61]: Why [ ]float32? They changed?

File: find.go [hunk 2]
```diff
@@ -26,21 +26,25 @@ type FindByID struct {
...
+			OriginalTitle    string   `json:"original_title,omitempty"` // Movie
+			OriginCountry    []string `json:"origin_country,omitempty"` // TV
+			Overview         string   `json:"overview"`
+			Popularity       float32  `json:"popularity"`
+			PosterPath       string   `json:"poster_path"`
+			ReleaseDate      string   `json:"release_date,omitempty"` // Movie
+			Title            string   `json:"title,omitempty"`        // Movie
+			Video            bool     `json:"video,omitempty"`        // Movie
+			VoteAverage      float32  `json:"vote_average"`
+			// VoteCount        int64   `json:"vote_count"` // FIXME: -> float32
```
Comment (cyruzin) [inline find.go:47]: image

**Comment by cyruzin** [comment]
```
package main

import (
	"fmt"

	tmdb "github.com/cyruzin/golang-tmdb"
)

func main() {
	tmdbClient, err := tmdb.Init("9aca69849a23528a419aea463387945f")

	if err != nil {
		fmt.Println(err)
	}

	options := make(map[string]string)
	options["external_source"] = "imdb_id"
	options["language"] = "en-US"

	const keanuReevesID = "nm0000206"

	result, err := tmdbClient.GetFindByID(keanuReevesID, options)
	if err != nil {
		fmt.Println(err)
	}

	if result.PersonResults != nil {
		fmt.Println("ID: ", result.PersonResults[0].ID)
		fmt.Println("Name: ", result.PersonResults[0].Name)
		if result.PersonResults[0].Gender == 1 {
			fmt.Println("Gender: Female")
		} else if result.PersonResults[0].Gender == 2 {
			fmt.Println("Gender: Male")
		} else {
			fmt.Println("Gender: Unknow")
		}

		fmt.Println("")
		fmt.Println("Known for: ")
		for _, known := range result.PersonResults[0].KnownFor {
			fmt.Printf("%s (%s)\n", known.Title, known.ReleaseDate)
		}
	}
}
```


Results:

```
ID:  6384
Name:  Keanu Reeves
Gender: Male

Known for:
The Matrix (1999-03-30)
John Wick (2014-10-22)
John Wick: Chapter 2 (2017-02-08)
```

**Comment by rqphqel** [comment]
Try it with `nm0000199`.

**Comment by rqphqel** [comment]
Looks like older entries return floats instead of ints.
Same with [credits](https://api.themoviedb.org/3/credit/52fe4267c3a36847f801be91?api_key=9aca69849a23528a419aea463387945f).

You could still merge those changes as it would work for both cases until we find a proper solution.

**Comment by cyruzin** [comment]
> Try it with `nm0000199`.

I’ll investigate.

**Comment by cyruzin** [comment]
@rqphqel update your PR, use float32 instead.

**Comment by rqphqel** [comment]
https://www.themoviedb.org/talk/6026eebd6a3448003e155bbc

```jsonc
// https://api.themoviedb.org/3/credit/5edc52301dcb77001d01b769?api_key=9aca69849a23528a419aea463387945f

{
  "media": {
    "genre_ids": [], // array
    "origin_country": [] // array
  },
  "person": {
    "known_for": [
      {
        "genre_ids": {}, // object
        "origin_country": {} // object
      }
    ]
  }
}
```
Even worse:
```jsonc
// https://api.themoviedb.org/3/find/nm0951517?api_key=9aca69849a23528a419aea463387945f&external_source=imdb_id

{
  "person_results": [
    {
      "known_for": {} // object
    }
  ]
}
```

**Comment by cyruzin** [comment]
I'll merge the current PR, let's wait TMDb do something.

**Pull request merged**